src/FX.GrpcModels/Marshalling.fs

```diff
@@ -4,7 +4,55 @@ open System
 open System.Reflection
 open System.Text.Json
 
-open FsharpExchangeDotNetStandard.Redis.Serialization
+[<AutoOpen>]
+module Serialization =
+    open FsharpExchangeDotNetStandard
+    open System.Text.Json.Serialization
+
+    type MatchTypeConverter() =
```

Comment:
@webwarrior-ws don't move it here please, MATCH is business logic (about the exchange), but Marshalling.fs is generic serialization that doesn't reference business logic

Reply:
OK, then where should `MatchTypeConverter` go? It's clearly a part of serialization code.

Reply:
new file, called ModelSerialization.fs